packages/react/src/modal/modal.tsx

```diff
@@ -44,11 +44,21 @@ export function WalletModal({
 
   const current = walletRepo?.current;
 
+  const [stytchState, setStytchState] = useState<State>(
+    // @ts-ignore
```

Comment:
Why the ts-ignore?

Reply:
It for some reason doesn't register the stytchState field as part of the client object. This is temporary. Ideally we can get it solved before actually pushing